src/frontend/apps/desk/src/features/teams/components/TeamInfo.tsx

```diff
@@ -58,21 +58,25 @@ export const TeamInfo = ({ team }: TeamInfoProps) => {
       </Box>
       <Box
         className="p-s"
-        $gap="1rem"
+        $gap="2rem"
         $direction="row"
-        $justify="space-evenly"
+        $justify="start"
         $css={`border-top: 1px solid ${colorsTokens()['card-border']};`}
       >
-        <Text $size="s">
+        <Text $size="s" as="p">
           {t('{{count}} member', { count: team.accesses.length })}
         </Text>
```

Comment:
In plural + In English, this element is not translated. Any clue @AntoLC ?

Reply:
Ah yes I know why... it is because we don't translate in english, but actually we should in these cases. 
Even if the keys are in english they are still keys, to be corrected we must have a value bind to the key, in english we don't have. Don't know if it is clear, I will open a issue about it.

Reply:
https://github.com/numerique-gouv/people/issues/99